python_scripts/01_tabular_data_exploration.py

```diff
@@ -57,18 +58,17 @@
 # %% [markdown]
 # The column named **class** is our target variable (i.e., the variable which
 # we want to predict). The two possible classes are `<= 50K` (low-revenue) and
-# `> 50K` (high-revenue).
+# `> 50K` (high-revenue). The resulting prediction problem is therefore a binary
+# classification problem.
+# while we will use the other columns as input variables for our model.
 
 # %%
 target_column = 'class'
 adult_census[target_column].value_counts()
 
 # %% [markdown]
-# Note: classes are slightly imbalanced. Class imbalance happens often in
-# practice and may need special techniques for machine learning. For example in
-# a medical setting, if we are trying to predict whether patients will develop
```

Comment:
I would leave the medical example. I feel it gives an example where class imbalance happens without going too much on a tangent.